### PR TITLE
Enable skip & limits & hide logs

### PR DESCRIPTION
**Pull request by javierarce**
This PR does:

* Enable the row limit
* Remove debugging logs
* Enable skip button
* Prevents an exception when the map doesn't have a url template

### CODE REVIEW

**Comment by javierarce** [comment]
cc @javisantana

File: lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_dialog_view.js [hunk 1]
```diff
@@ -126,7 +126,11 @@ module.exports = BaseDialog.extend({
   _onSkipClick: function(e) {
     this.killEvent(e);
     this.model.set("disabled", true);
-    this.close(); // TODO: trigger a skip after the manual testing is finished
+
+    var skipPencanDialog = 'pecan_' + this.vis.tableMetadata().attributes.id;
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_dialog_view.js]: I guess I could use .get("id") here. I'll change it.

Comment (javisantana) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_dialog_view.js]: I'd use username too to avoid collision between accounts

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5134/))

File: lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_dialog_view.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/pecan_dialog_view.js]: Right, I'll fix it.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5136/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5137/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5138/))

**Pull request merged**